docs/operate/config-reference.mdx

```diff
@@ -1075,6 +1075,50 @@ Because each VM supports a different signer type, the default signer can only su
 </TabItem>
 </Tabs>
 
+## defaultRpcConsensusType
+
+:::info
+
+**The consensus type currently only applies to EVM chains.**
+
+Connections to non-EVM chains only support a single RPC url. If more than one RPC url is specified in [rpcUrls](#chainschain_namerpcurls), only the first one will be used.
+
+:::
+
+**Description:** How to communicate with the provider
```

Comment:
I would mention that `defaultRpcConsensusType` is only used when building EVM providers: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/2d84c0f961f25c5c9b6580247cd1ae28b02a9fdd/rust/main/hyperlane-base/src/settings/parser/connection_parser.rs#L468

Reply:
https://github.com/hyperlane-xyz/v3-docs/pull/364/commits/4ecee1d0e272a4ab312879655620370f79dc614c#diff-7eaa4255dc3e4bcf29a6da37e2002d561b18362706333bb30df419de86f1ea51R1082-R1084
this already mentions that, any additional notes you would like, or is this not that clear at first glance?

Reply:
updated in 163547b5ddb345a7c7b90e69be1d609c571ef8c4